manuscript/chapter2.md

```diff
@@ -481,7 +481,8 @@ class ExplainBindingsComponent extends Component {
 }
 ~~~~~~~~
 
-You should avoid it too, because it will clutter your constructor over time. The constructor is only there to instantiate your class with all its properties. That's why the business logic of class methods should be defined outside of the constructor.
+Avoid this method as well, as it will clutter your constructor over time. The constructor is only there to instantiate your class with all its properties, so the business logic of class methods should be defined outside the constructor.
+
```

Comment:
remove extra line

Reply:
maybe "Avoid this approach as well", because method is used for class methods here and may be misleading